var/spack/repos/builtin/packages/otf2/package.py

```diff
@@ -24,6 +24,8 @@ class Otf2(AutotoolsPackage):
     version('1.3.1', sha256='c4605ace845d89fb1a19223137b92cc503b01e3db5eda8c9e0715d0cfcf2e4b9')
     version('1.2.1', sha256='1db9fb0789de4a9c3c96042495e4212a22cb581f734a1593813adaf84f2288e4')
 
+    patch('collective_callbacks.patch', when='@2.1:2.2')
```

Comment:
Can you add a comment above this patch with provenance (the issue you linked in the description is fine) and a brief description of why it is needed?

Reply:
Absolutely.